runtime/lua/man.lua

```diff
@@ -21,11 +21,21 @@ local function system(cmd, silent, env)
   local done = false
   local exit_code
 
+  -- we use the `env` command here rather than the env option to vim.loop.spawn since spawn will
+  -- completely overwrite the environment when we just want to modify the existing one
```

Comment:
Replacing the environment is ok on the majority of systems. Probably need to mention we only need to do this for nix.